book/routing.rst

```diff
@@ -133,15 +39,23 @@ else. Take the following HTTP request for example:
 The goal of the Symfony routing system is to parse this URL and determine
 which controller should be executed. The whole process looks like this:
 
-#. The request is handled by the Symfony front controller (e.g. ``app.php``);
+* Each request executes a single front controller file;
+
+* Front controller's only job is to initialize Symfony's engine (called the
+  :term:`Kernel`) and pass it a ``Request`` object to handle;
+
+* The Symfony core asks the router to inspect the request;
 
-#. The Symfony core (i.e. Kernel) asks the router to inspect the request;
+* The router matches the incoming URL to a specific route and returns
+  information about the route, including the controller that should be
+  executed;
 
-#. The router matches the incoming URL to a specific route and returns information
-   about the route, including the controller that should be executed;
+* The correct controller from the matched route is executed and the code
+  inside the controller creates and returns the appropriate ``Response``
+  object;
 
-#. The Symfony Kernel executes the controller, which ultimately returns
-   a ``Response`` object.
+* The HTTP headers and content of the ``Response`` object are sent back
+  to the client.
```

Comment:
Ah, I see we basically have this duplicated in the controller chapter. Let's include it in one spot only - probably it makes more sense here.